requirements.txt

```diff
@@ -1,24 +1,24 @@
 beautifulsoup4
 fake_useragent
 imageio
-keras
+keras ; python_version < '3.12'
 lxml
 matplotlib
 numpy
 opencv-python
 pandas
 pillow
-projectq
+# projectq  # uncomment once quantum/quantum_random.py is fixed
 qiskit ; python_version < '3.12'
 qiskit-aer ; python_version < '3.12'
 requests
 rich
-scikit-fuzzy
+# scikit-fuzzy  # uncomment once fuzzy_logic/fuzzy_operations.py is fixed
 scikit-learn
 statsmodels
 sympy
 tensorflow ; python_version < '3.12'
 texttable
```

Comment:
```suggestion
```
I don't think texttable is used in any file, but I might've missed a usage in my search. @cclauss, could you confirm?

Reply:
@cclauss?